tn_api_v2_core.yml

```diff
@@ -0,0 +1,956 @@
+                    description: Key corresponding to above fare_info_ref also required to make a travelport fare rule request. *Travelport Only*
+                    example: "gws-eJxNjrEOwyNVDF8mut2YNjOSLC0PGOVC1DFER+f/P6EGWnmR8+Fm2QwgqakXVhX8NOIYyob4KCMHP+Fhg/FUEhr8dYvSGpczb+pRPPmQ0lqh2fGbTGmX22TMhu6QsEDZh7+8zFbJzstCgrUazLFCi40XxudPWAb/2bY1iXKdiaXgIeOkPoWIr6Q=="
+                baggage:
+                    type: object
+                    description: Contains baggage information.
+                    properties:
+                        number_of_pieces:
+                            type: integer
+                            example: 0
+                brands:
+                    $ref: '#/components/schemas/DatasourceBrandDetails'
+                price_breakdown_with_passenger_type_code:
+                    type: array
+                    description: Displays the pricing for each passenger type
+                    items:
+                        $ref: '#/components/schemas/DatasourcePriceBreakdownDetails'
+                plating_carrier:
+                    type: string
+                    description: Plating carrier. Example = "WS"
+                    example: "WS"
+                intermediate_stops:
+                    type: array
+                    description: Direct flight between two points with no change in flight number
+                    items:
+                        $ref: '#/components/schemas/DatasourceLegDetails'
+                price:
+                    type: number
+                    description: Price of the segment (fare and tax included).
+                    example: 123.45
+                e_ticketability:
+                    type: boolean
+                    description: E-tickable tickets
+                    example: true
+                cancel_penalty:
+                    type: object
+                    description: Information about cancellation
+                    properties:
+                        amount:
+                            type: number
+                            description: Cancellation penalty amount
+                            example: 0
+                        percentage:
+                            type: number
+                            description: Cancellation penalty percentage
+                            example: 100
+                change_penalty:
+                    type: object
+                    description: Information about changing.
+                    properties:
+                        amount:
+                            type: float
+                            description: Change penalty amount
+                            example: 78
+                        percentage:
+                            type: float
+                            description: Change penalty percentage
+                            example: 0
+
+        MinimalDatasourceLegDetails:
+            description: A list of legs for this segment
```

Comment:
Minor - I think this description could be improved a bit, as a list of legs is a little vague. Maybe

A list of required info for each leg of this segment.